src/deprecated.js

```diff
@@ -287,29 +89,75 @@ pc.string.format = function (s) {
     return s;
 };
 
-pc.string.startsWith = function (s, subs) {
+string.startsWith = function (s, subs) {
     // #ifdef DEBUG
     console.warn("DEPRECATED: pc.string.startsWith is deprecated. Use String#startsWith instead.");
     // #endif
     return s.startsWith(subs);
 };
 
-pc.time = {
+export var time = {
     now: pc.now,
     Timer: pc.Timer
```

Comment:
Removing pc namespace and adding `import {now, Timer} from "../src/core/time";` fixes this one

Reply:
Fixed